test/e2e/harness/src/SessionTokenLogin.tsx

```diff
@@ -29,7 +29,8 @@ const SessionTokenLogin: React.FC = () => {
     })
     .then((res: AuthTransaction) => {
       setSessionToken(res.sessionToken);
-      oktaAuth.signInWithRedirect({sessionToken: res.sessionToken});
+      oktaAuth.setOriginalUri('/');
```

Comment:
not necessary if `restoreOriginalUri` uses `/` as a default  in the case of null